src/govuk/i18n.mjs

```diff
@@ -82,7 +82,10 @@ I18n.prototype.replacePlaceholders = function (translationString, options) {
 
       // If a user has passed `false` as the value for the placeholder
       // treat it as though the value should not be displayed
-      if (placeholderValue === false) {
+      if (placeholderValue === false || (
+        typeof placeholderValue !== 'number' &&
+        typeof placeholderValue !== 'string')
+      ) {
```

Comment:
@36degrees Yeah it's to satisfy the JSDoc

```js
/**
 * @returns {string} The translation string to output, with ${} placeholders replaced
 */
```

We dealt with the following:

`{false}` → Empty string
`{number}` → Formatted to strings

But the compiler knows we say `{Object<string, unknown>}` can be passed to those `.t()` functions

Rather than adhere to the types, I'll double check on whether the types are correct 👍

Reply:
Sounds to me that what we accept as an argument is not so `unknown`, so shouldn't we properly document the restriction to the options being `Object<string, boolean|number|string>`? Would that spare us these `if`s?

Reply:
Oh, forgot the `mergeConfigs` in the mix, that makes sense that we need `unknown` then. Thanks for the explanation 😄

Reply:
Regarding changing the behaviour (in the original message), this is part of the `I18n` class, which we consider private for now so I don't think that'd be a breaking change there.